src/lib/holocene/pagination.svelte

```diff
@@ -206,7 +206,7 @@
     setActiveRowIndex={store.setActiveRowIndex}
   />
   <nav
-    class={`flex ${
+    class={`flex text-primary ${
```

Comment:
Is `text-primary` needed in both places? Should we be setting it in the `FilterSelect` instead?

Reply:
Oh that's a really great idea! Still getting used to this codebase for sure.